spec/try_spec.rb

```diff
@@ -35,6 +35,14 @@
     it { expect(result[2][:status]).to eq :failed }
   end
 
+  context 'try with cd to invalid directory - multiple times' do
+    let(:request) { struct query: 'cat nonexistent_directory', goal: goal }
+    let(:results) { 5.times.map { hook.run! hook.compile(request) } }
```

Comment:
I changed it from `Array.new(5)` to `5.times.map` which i find more intention revealing